src/index.js

```diff
@@ -170,6 +169,10 @@ class MiniCssExtractPlugin {
           const renderedModules = Array.from(chunk.modulesIterable).filter(
             (module) => module.type === MODULE_TYPE
           );
+          const { filename } = this.options;
+          const filenameTemplate =
```

Comment:
You can pass a function as filenameTemplate.

Reply:
Can you elaborate more on this?

Reply:
oh, do you mean a separate option as `filenameTemplate` instead of `filename` for both?

Reply:
:man_facepalming: I see what you meant. fixed

Reply:
Switched to filenameTemplate